README.adoc

```diff
@@ -1956,6 +1956,30 @@ travel_to(Time.current.to_time)
 freeze_time
 ----
 
+== Performance
```

Comment:
I felt that the category "Performance" is a bit too broad. I’m not entirely confident, but after looking over the categories, it seems to me that adding "Jobs" under "[Mailers](https://rails.rubystyle.guide/#mailers)" would be a good fit. Strictly speaking, it covers not only Active Job but also Action Mailer, but I made this judgment because the link URL itself includes "jobs".

@rubocop/style-guide-editors If there is a better approach, please suggest it.